pkg/controller/command/errors.go

```diff
@@ -0,0 +1,82 @@
+/*
+ *
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * /
+ *
+ */
+
+package command
+
+// Type is command error type.
+type Type int32
+
+const (
+	// ValidationError is error type for command validation errors.
+	ValidationError Type = iota
+
+	// ExecuteError is error type for command execution failure.
+	ExecuteError Type = iota
+)
+
+// Code is the error code of command errors.
+type Code int32
+
+const (
+	// UnknownStatus default error code for unknown errors.
+	UnknownStatus Code = iota
+)
+
+// Group is the error groups.
+// Note: recommended to use [0-9]*000 pattern for any new entries
+// Example: 2000, 3000, 4000 ...... 25000
+type Group int32
+
+const (
+	// Common error group for general command errors
+	Common Group = 1000
+
+	// DIDExchange error group for DID exchange protocol command errors
+	DIDExchange Group = 2000
```

Comment:
We shouldn't be referencing specific protocols.

Reply:
these are specific to command types, I will fix the comment.

Reply:
fixed

Reply:
My point is actually that DIDExchange nor any other specific protocol/message should not appear in this file.

Reply:
I guess this is existing code, but I don't really understand why we didn't use a namespace string defined by individual command packages.